.circleci/config.yml

```
@@ -711,6 +721,25 @@ jobs:
       - persist_to_workspace: *artifacts_executables
       - gitter_notify_failure_unless_pr
 
+  b_ubu_static_arm:
+    <<: *base_ubuntu2004_arm_large
+    environment:
+      TERM: xterm
+      MAKEFLAGS: -j 5
+      CMAKE_OPTIONS: -DCMAKE_BUILD_TYPE=Release -DUSE_Z3_DLOPEN=ON -DSTRICT_Z3_VERSION=OFF -DUSE_CVC4=OFF -DSOLC_STATIC_STDLIBS=ON
```

In CI we always want to run with `-DSTRICT_Z3_VERSION=ON`
```suggestion
      CMAKE_OPTIONS: -DCMAKE_BUILD_TYPE=Release -DUSE_Z3_DLOPEN=ON -DUSE_CVC4=OFF -DSOLC_STATIC_STDLIBS=ON
```